resources/main.mjs

```diff
@@ -195,12 +195,16 @@ class MainBenchmarkClient {
         }
         document.getElementById("metrics-results").innerHTML = html;
 
-        const jsonData = JSON.stringify(this._measuredValuesList);
-        const blob = new Blob([jsonData], { type: "application/json" });
-        const url = URL.createObjectURL(blob);
-        const link = document.getElementById("download-json");
-        link.href = url;
-        link.setAttribute("download", `speedometer_3-${new Date().toISOString()}.json`);
```

Comment:
Why are we deleting the feature to download JSON file? That seems useful.

Reply:
It's not deleting the feature, this diff is due to a variable rename (now jsonLink)